### ISSUE TITLE
Minor issues when working on QHD screen in Windows

### ISSUE THREAD

**Issue by rwbaer** [makehumancommunity/makehuman#186]
Windows 10.  Source run 1.2.1 alpha
I have a laptop with QHD and have encountered a couple of minor behaviors when running MH with Use HDPI checked.  When first time restart, the splash screen appears and disapears and is replace by a blank white panel.  This eventually populates with the program, but it    takes lone enough that the program appears crashed even on a fast machine.  I'm not sure if this is the normal plugin loading or something else, but there is no indication that anything is happening.

During the first start up, the default size of the program is a little larger than the screen space I have right after HPDI is restarted (covers my menu bar).  Once I fix this once, it is okay on subsequent runs.

**Unrelated**: Sometime during the start-up I see a 1.2.0 flash on the screen so there is at least one other place to fix the version.  Is it hard coded in multiple places or is the splash screen just an updated image and the code hasn't been fixed?

**Comment by rwbaer** [comment]
Windows 21H.22000.348  MH-20211213 alpha (HEAD:af7a054b) - Use HDPI checked
graphics card - Nvidia 3080
QT
As I have moved forward, I have not been encountering the white screen that started this report.  There have been Windows updates 11, so maybe this is a transient issue.  I am also moved on to the current MH nightly rather than 1.2 stable for this report.

I am encountering a new set of problems that seem screen related.  The configuration is that the main laptop has a QHD screen.  I have two side monitors that are HD screens.  If I drag MH onto a side screen, it rearranges its size, but something goes wrong with the resizing of the main modeling viewport.  It stays large and off-center with the top left corner pinned. See screenshot

![SideScreen](https://user-images.githubusercontent.com/3155051/146014220-f76b997e-c7a2-4ded-b98c-104dc7a41cf3.png)

In addition, after clicking on settings, I was able to select utilities, help, and community tabs, but not files, modeling geometries, materials, pose/animate, or rendering.  If I then drag MH back on to my center (QHD-main laptop) scree, the tabs become selectable again.

The dragging back and forth process is associated with a long "not responding" period.  When dragged again to the side screen, I get some icons (save,export load) that are written over the tabs up to but not including settings.  It is like part of the QT code is resizing properly and part of it is not as I drag from QHD to HD screen. The shot below is a 3rd drag, the icons are not covering the upper left screen, but it is clear there is other stuff being drawn there.
![image](https://user-images.githubusercontent.com/3155051/146017728-838677d1-370e-4cfa-accc-b0cf23e5b8e6.png)



I'll finish off by saying during the process of dragging back and forth,  I did encounter some additional rectangular white screen patches that remind me of the original issue, just not at start-up.  They seem associated with redrawing as I drag from the HD to QHD screen.

I'm guessing this issue is not very actionable at this point, but it seems worth documenting to see if it happens to anyone else as Windows 11 rolls out.
